pkgs/top-level/all-packages.nix

```diff
@@ -19040,6 +19040,8 @@ with pkgs;
 
   redis-desktop-manager = libsForQt56.callPackage ../applications/misc/redis-desktop-manager { };
 
+  robo3t = callPackage ../applications/misc/robo3t { };
+  
   robomongo = callPackage ../applications/misc/robomongo { };
```

Comment:
Is there a need for the old version?

Reply:
I am new to nixos so I don't know if there is a way to mark an attribute as deprecated. Should the old version be left there to avoid breaking people's configs that use it? Or is it standard practice to just remove something like that?

Reply:
Just remove the old package and add an entry to `pkgs/top-level/aliases.nix`.

Reply:
OK thanks, amended commit.